aws/scripts/py/s3_sandbox_configuration.py

```
@@ -0,0 +1,65 @@
+        },
+        "ErrorDocument": {
+            "Key": "error.html",
+        },
+    }
+    json_string = json.dumps(config, indent=4)
+
+    try:
+        with open(output_path, 'w') as file:
+            file.write(json_string)
+    except OSError as e:
+        raise RuntimeError(f"Failed to write website configuration: {e}") from e
```

Ruff

Avoid specifying long messages outside the exception class

(TRY003)